src/content/docs/backend/API/unit-tutorial_streams.md

```diff
@@ -0,0 +1,134 @@
+---
+title: Doubtfire API
+---
+
+# List of Doubtfire
+
+# API: Units (Page 1)
```

Comment:
Please change the title here to the new category you made, rather than using Units (Page 1), the formatting below this also is dissimilar to other pages on the site.